process/logsevents/fungibleESDTProcessor.go

```diff
@@ -64,21 +75,27 @@ func (fep *fungibleESDTProcessor) processEventOnSenderShard(event coreData.Event
 	})
 }
 
-func (fep *fungibleESDTProcessor) processEventDestination(event coreData.EventHandler, accounts data.AlteredAccountsHandler, selfShardID uint32) string {
-	topics := event.GetTopics()
+func (fep *fungibleESDTProcessor) processEventDestination(args *argsProcessEvent, senderShardId, selfShardID uint32) string {
```

Comment:
`senderShardID` * also, use param type near each param

Reply:
...or use another argument DTO in order to avoid confusions when calling this function

Reply:
done.